runtime/master/src/main/java/org/apache/nemo/runtime/master/scheduler/SimulatedTaskExecutor.java

```diff
@@ -106,15 +106,19 @@ private long calculateExpectedTaskDuration(final Task task) {
       .filter(s -> s.get("properties").get("irDag").get("edges").size()
         == stageIRDAG.getEdges().size())  // same # of edges.
       .filter(s -> s.get("properties").get("executionProperties")
-        .get("org.apache.nemo.common.ir.vertex.executionproperty.ParallelismProperty").asInt(0)
-        == task.getPropertyValue(ParallelismProperty.class).orElse(0))  // same parallelism.
+        .get("org.apache.nemo.common.ir.vertex.executionproperty.ParallelismProperty").asInt(1)
+        == 32)  // sampling vertices have parallelism of 32
```

Comment:
It would be even better if 32 is a global variable so that we don't forget to update this number when later we decide to change the parallelism of sampling vertex.